builtins/web/file.cpp

```diff
@@ -122,45 +107,47 @@ bool File::lastModified_get(JSContext *cx, unsigned argc, JS::Value *vp) {
     return api::throw_error(cx, api::Errors::WrongReceiver, "lastModified get", "File");
   }
 
-  auto lastModified = JS::GetReservedSlot(self, static_cast<size_t>(File::Slots::LastModified)).toInt32();
+  auto lastModified =
+      JS::GetReservedSlot(self, static_cast<size_t>(Slots::LastModified)).toNumber();
   args.rval().setNumber(lastModified);
   return true;
 }
 
-bool File::init_last_modified(JSContext *cx, HandleObject self, HandleValue initv) {
-  bool has_last_modified = false;
-
-  JS::RootedValue init_val(cx, initv);
-  JS::RootedObject opts(cx, init_val.toObjectOrNull());
-
-  if (!opts) {
-    return true;
+bool File::init(JSContext *cx, HandleObject self, HandleValue fileBits, HandleValue fileName, HandleValue opts) {
```

Comment:
Can you add a doc comment with a link to the algorithm implemented here? See some of the other builtins for how I usually format those.